streaming/streaming.go

```diff
@@ -463,7 +465,8 @@ func createNotificationOnBucket(ctx context.Context, storageClient *storage.Clie
 
 // LaunchStream populates the parameters from the streaming config and triggers a stream on Cloud Datastream.
 func LaunchStream(ctx context.Context, sourceProfile profiles.SourceProfile, dbList []profiles.LogicalShard, projectID string, datastreamCfg DatastreamCfg) error {
-	fmt.Println("Launching stream ", fmt.Sprintf("projects/%s/locations/%s", projectID, datastreamCfg.StreamLocation))
+	projectNumber := GetProjectNumber(ctx, fmt.Sprintf("projects/%s", projectID))
+	fmt.Println("Launching stream ", fmt.Sprintf("%s/locations/%s", projectNumber, datastreamCfg.StreamLocation))
```

Comment:
+1

Reply:
Makes Complete sense.  Incorporated the change (w/o loosing the current flexibility that potentially allows different ds in different projects in case cx has different mysql shards in vms across projects)